benchmark-utils_test.go

```diff
@@ -224,21 +201,62 @@ func runGetObjectBenchmark(b *testing.B, obj ObjectLayer, objSize int) {
 
 }
 
-// randomly picks a character and returns its equivalent byte array.
-func getRandomByte() []byte {
-	const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
-	// seeding the random number generator.
-	rand.Seed(time.Now().UnixNano())
-	var b byte
-	// pick a character randomly.
-	b = letterBytes[rand.Intn(len(letterBytes))]
-	return []byte{b}
+const alphabets = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+// Returns random character as byte from letterBytes.
+func getRandByte() byte {
+	return alphabets[rand.Intn(len(alphabets))]
 }
 
-// picks a random byte and repeats it to size bytes.
-func generateBytesData(size int) []byte {
```

Comment:
I've used this function for other tests, it'll break them if you remove. Can you move this from here and add to test-utils.go .